airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/common.json

```diff
@@ -93,6 +93,12 @@
     "or": "或"
   },
   "filter": "筛选",
+  "filters": {
+    "logicalDateFrom": "从逻辑日期",
+    "logicalDateTo": "到逻辑日期",
+    "runAfterFrom": "从运行之后",
+    "runAfterTo": "到运行之后"
```

Comment:
```suggestion
    "runAfterTo": "到运行之前"
```

Reply:
might need @potiuk @Fortytwoo @gyli  's confirmation on these. but can be done in another PR